### PR TITLE
fix compaction speedup for marked files ut

### PR DESCRIPTION
**Pull request by SGZW**

### CODE REVIEW

File: db/column_family_test.cc [hunk 1]
```diff
@@ -3091,9 +3100,9 @@ TEST_P(ColumnFamilyTest, CompactionSpeedupForMarkedFiles) {
   ASSERT_EQ(kParallelismLimit, dbfull()->TEST_BGCompactionsAllowed());
```
Comment (SGZW) [inline db/column_family_test.cc:3100]: When I run UT(CompactionSpeedupForMarkedFiles) locally, this line often fails to pass, and the reason is the current test case will share the same env with another case that may have already increased the number of background threads. So I make a simple fix

**Comment by SGZW** [comment]
@ajkr PTAL, thanks!

File: db/column_family_test.cc [hunk 2]
```diff
@@ -3067,12 +3067,21 @@ TEST_P(ColumnFamilyTest, CompactionSpeedupForMarkedFiles) {
   WaitForCompaction();
   AssertFilesPerLevel("0,1", 0 /* cf */);
 
+  // We should calculate the limit by obtaining the maximum value between the
+  // number of env background threads and kParallelismLimit, because the current
+  // test case will share the same env with another case that may have already
+  // increased the number of background threads.
+  const auto limit = std::max(env_->GetBackgroundThreads(Env::Priority::LOW),
```
Comment (cbi42) [inline db/column_family_test.cc:3074]: Can we just use `env_->GetBackgroundThreads(Env::Priority::LOW)` here? We cannot schedule more than the limit anyway.

Comment (SGZW) [reply db/column_family_test.cc:3074]: ok, i will fix it

Comment (SGZW) [reply db/column_family_test.cc]: done

**Pull request closed without merging**